cdap-integration-test/src/main/java/io/cdap/cdap/test/IntegrationTestBase.java

```diff
@@ -397,8 +397,12 @@ private void doClear(NamespaceId namespace, boolean deleteNamespace) throws Exce
     }
 
     // delete all apps in the namespace
+    HashMap<String, String> uniqueApps = new HashMap<>();
     for (ApplicationRecord app : getApplicationClient().list(namespace)) {
-      getApplicationClient().delete(namespace.app(app.getName(), app.getAppVersion()));
```

Comment:
On a second thought, we can also call `getApplicationClient().deleteApp` and ignore `ApplicationNotFoundException` wdyt??

Reply:
yea. Since it's cleanup, I guess we don't need to care about application not found